rn-cli-inside-gb.config.js

```diff
@@ -26,7 +24,6 @@ const alternateRoots = [ path.resolve( __dirname, '../node_modules' ) ].concat(
 );
 
 module.exports = {
-	extraNodeModules: Object.assign( enm, wppackages ),
```

Comment:
I'm not sure if `wppackages` is used in this case but I think it's better to only remove `enm` and keep it like `extraNodeModules: wppackages`

Reply:
Good catch, I missed that!